sui-execution/latest/sui-adapter/src/programmable_transactions/context.rs

```diff
@@ -963,6 +964,7 @@ mod checked {
             function_name: &IdentStr,
             ty_args: Vec<Type>,
             args: Vec<impl Borrow<[u8]>>,
+            tracer: Option<&mut MoveTraceBuilder>,
```

Comment:
Should the tracer be part of the `ExecutionContext`?

Reply:
Not sure how useful it would be as I am not sure it will be used anywhere else other than runtime's `execute_function_bypass_visibility` function